### PR TITLE
Inline a once-used one-line function

### PR DESCRIPTION
**Pull request by mmcco**
Already done in OpenBSD.

### CODE REVIEW

File: linenum.c [hunk 1]
```diff
@@ -217,6 +207,7 @@ longish(void)
 	if (loopcount >= 0 && ++loopcount > 100) {
 		loopcount = 0;
 		if (time(NULL) >= startime + LONGTIME) {
+			ierror("Calculating line numbers", NULL_PARG);
 			longloopmessage();
```
Comment (Pyrohh) [inline linenum.c]: Still need to remove the call site :)

**Comment by mmcco** [comment]
@Pyrohh Thanks. Reasons not to manually replay diffs on Christmas morning.   :-)

**Comment by gdamore** [comment]
I'm gonna rebase these into a single commit.

**Comment by gdamore** [comment]
Merged/rebased together:

commit 5b191cffb2813639abfc489be402c0a7eced4fce
Author: Michael McConville <mmcco@mykolab.com>
Date:   Fri Dec 25 13:01:38 2015 -0500

    Inline a once-used one-line function

**Pull request closed without merging**